lib/sequel-fixture.rb

```diff
@@ -47,7 +46,7 @@ def initialize(fixture = nil, connection = nil, option_push = true)
   def load(fixture)
     raise LoadingFixtureIllegal, "A check has already been made, loading a different fixture is illegal" if @checked
 
-    Fast.dir("#{fixtures_path}/#{fixture}").files.each do |file|
+    Dir.entries("#{fixtures_path}/#{fixture}").reject {|f| f =~ /^\./} .each do |file|
```

Comment:
Added the `reject` to skip `.` and `..` files.  Then noticed that we're splitting on `.` anyway so probably want to skip any file starting with `.`